### DISCUSSION TITLE
NodeJS 14 or 16?

### DISCUSSION THREAD

**Discussion by Bunker56** [PhlexPlexico/G5API#124] [Q&A] [answered]
Node 14 or 16? Or is there no difference?

**Comment by PhlexPlexico** [comment, marked as answer]
I remember awhile back bumping the latest node version to include some optional chaining parameters. Anything above Node 14 should work just fine!

**Reply by Bunker56** [reply]
OK. I'll chance it. :)

**Reply by Bunker56** [reply]
works great in Docker

**Comment by Bunker56** [comment]
SHELL ["/bin/bash", "--login", "-c"]
RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.38.0/install.sh | bash
RUN nvm install node
